src/AltinnCore/Common/Models/DeploymentResponse.cs

```diff
@@ -0,0 +1,24 @@
+namespace Common.Models
+{
+    /// <summary>
+    /// Deployment response
+    /// </summary>
+    public class DeploymentResponse
+    {
+        /// <summary>
+        /// Success status
+        /// </summary>
+        public bool Success { get; set; }
```

Comment:
Unsure if this is acctually needed, it can be there for now. But it should probably be renamed or removed when we figure out what the frontend needs